deploy-v2.js

```diff
@@ -0,0 +1,162 @@
+const {
+  Web3Connection,
+  ERC20,
+  BountyToken,
+  Network_v2,
+  NetworkFactoryV2,
+  toSmartContractDecimals,
+  Network_Registry
+} = require("@taikai/dappkit");
+const { nativeZeroAddress } = require("@taikai/dappkit/dist/src/utils/constants");
```

Comment:
You can import this from `@taikai/dappkit` by importing `import {Defaults} from '@taikai/dappkit'` and then use it as `Defaults.nativeZeroAddress`